test/test_macros.jl

```diff
@@ -1510,16 +1520,16 @@ end
 
 function test_MA_Zero_objective()
     model = Model()
-    @test @objective(model, Min, sum(i for i in 1:0)) === 0.0
+    @test @objective(model, Min, sum(i for i in 1:0)) === false
     return
 end
 
 function test_MA_Zero_expression()
     model = Model()
-    @test @expression(model, sum(i for i in 1:0)) === 0.0
+    @test @expression(model, sum(i for i in 1:0)) === false
     @expression(model, expr[j = 1:2], sum(i for i in j:0))
-    @test expr == [0.0, 0.0]
-    @test expr isa Vector{Float64}
+    @test expr == [false, false]
+    @test expr isa Vector{Bool}
```

Comment:
Yeah I think this is wrong, and arguably breaking. I know a bunch of people use this behavior (e.g., PowerModels and PowerSimulations).